wpilibj/src/main/java/edu/wpi/first/wpilibj/command/InstantCommand.java

```diff
@@ -42,8 +44,60 @@ public InstantCommand(String name, Subsystem requirement) {
     super(name, requirement);
   }
 
+  /**
+   * Creates a new {@link InstantCommand InstantCommand}.
+   * @param func the function to run on initialize
+   */
+  public InstantCommand(Runnable func) {
+    m_func = func;
+  }
+
+  /**
+   * Creates a new {@link InstantCommand InstantCommand}.
+   * @param name the name for this command
+   * @param func the function to run on initialize
+   */
+  public InstantCommand(String name, Runnable func) {
+    super(name);
+    m_func = func;
+  }
+
+  /**
+   * Creates a new {@link InstantCommand InstantCommand}.
+   * @param requirement the subsystem this command requires
+   * @param func        the function to run on initialize
+   */
+  public InstantCommand(Subsystem requirement, Runnable func) {
+    super(requirement);
+    m_func = func;
+  }
+
+  /**
+   * Creates a new {@link InstantCommand InstantCommand}.
+   * @param name        the name for this command
+   * @param requirement the subsystem this command requires
+   * @param func        the function to run on initialize
```

Comment:
These comments aren't consistent with the C++ ones (and vice versa). We try to keep them identical except for javadoc-specific things like `{@link}`.